base_layer/core/src/validation/aggregate_body/aggregate_body_chain_validator.rs

```diff
@@ -191,9 +191,6 @@ fn check_validator_node_registration_utxo(
             });
         }
 
-        // TODO(SECURITY): Signing this with a blank msg allows the signature to be replayed. Using the commitment
-        //                 is ideal as uniqueness is enforced. However, because the VN and wallet have different
-        //                 keys this becomes difficult. Fix this once we have decided on a solution.
```

Comment:
😬

Reply:
haha yeah, they are now marked as an issue on the dan project to figure out what exactly they want to add there